.github/workflows/ci.yml

```diff
@@ -16,6 +16,9 @@ jobs:
       - run: "moon ci --color"
         env:
           PUBLIC_LANGGRAPH_API_URL: 'http://127.0.0.1:2024'
+          # Required for auth.ts env validation
+          AUTH_DESCOPE_ID: 'bogus_id'
+          AUTH_DESCOPE_SECRET: 'bogus_secret'
```

Comment:
Biggest part of my prior feedback is so we can remove this and keep our setup as simple as possible.